infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/secure/jwtproxy/JwtProxyConfigBuilder.java

```diff
@@ -81,13 +106,17 @@ public String build() {
                   Collections.singleton(
                       new RegistrableComponentConfig()
                           .withType("static")
-                          .withOptions(ImmutableMap.of("iss", "wsmaster"))))
+                          .withOptions(ImmutableMap.of("iss", issuer))))
               .withNonceStorage(new RegistrableComponentConfig().withType("void"));
 
       if (!verifierProxy.excludes.isEmpty()) {
         verifierConfig.setExcludes(verifierProxy.excludes);
       }
 
+      if (authPageUrl != null) {
+        verifierConfig.setAuthUrl(authPageUrl.toString());
```

Comment:
Just to clarify: What will do JwtProxy if auth url is absent? Just send 403?

Reply:
yes